src/main.cpp

```diff
@@ -697,7 +701,12 @@ int main(int Argc, char *Argv[]) {
             errorHandler.AddError(
                 "Option \"link\" can't be used in compilation mode. Use \"ispc link --help\" for details");
         } else if (!strcmp(argv[i], "--support-matrix")) {
-            g->target_registry->printSupportMatrix();
+            std::vector<std::string> missedFiles;
+            g->target_registry->printSupportMatrix(missedFiles);
```

Comment:
It was more like ad-hoc debugging functionality for checking correctness. I outlined missedFiles checking logic to a separate function and introduced `--check-bitcode-libs` option for calling it.

Reply:
This solution does make more sense. But why still doing that only on request. So, it basically verifies integrity of the installation, right?

I'm wondering if we should do that unconditionally on every run? It's an open question, I don't know if should do that or now.

Also, are you introducing any ENV variable or configuration mechanisms to locate these libs? It's also an open question. There was a statement in the design doc that this might be a way to distro owners to build ISPC, but the location might be different in every distro. Again, it's not a call to action, but would be good to understand this more clearly.